src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -54,37 +55,54 @@ async fn test_sysmon_log_e2e(ctx: &mut E2eTestContext) -> Result<(), Box<dyn std
     let raw_logs_scanner_handle = KafkaTopicScanner::new(
         ConsumerConfig::with_topic("raw-logs"),
         Duration::from_secs(60),
+        Envelope::new(
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            RawLog::new(test_fixtures::single_sysmon_event()),
+        ),
     )
     .scan_for_tenant(tenant_id, 36, |_log: RawLog| true)
-    .await; // this blocks for 10s
+    .await;
 
     let generated_graphs_scanner_handle = KafkaTopicScanner::new(
         ConsumerConfig::with_topic("generated-graphs"),
         Duration::from_secs(60),
+        Envelope::new(
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            GraphDescription::new(),
+        ),
     )
-    .scan_for_tenant(tenant_id, 360, |_graph: GraphDescription| true)
-    .await; // this blocks for 10s
+    .scan_for_tenant(tenant_id, 36, |_graph: GraphDescription| true)
+    .await;
 
     let node_identifier_scanner_handle = KafkaTopicScanner::new(
         ConsumerConfig::with_topic("identified-graphs"),
         Duration::from_secs(60),
+        Envelope::new(
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            IdentifiedGraph::new(),
+        ),
     )
-    .scan_for_tenant(tenant_id, 360, |_graph: IdentifiedGraph| true)
-    .await; // this blocks for another 10s
+    .scan_for_tenant(tenant_id, 36, |_graph: IdentifiedGraph| true)
+    .await;
 
     let graph_merger_scanner_handle = KafkaTopicScanner::new(
         ConsumerConfig::with_topic("merged-graphs"),
         Duration::from_secs(60),
+        Envelope::new(
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            Uuid::new_v4(),
+            MergedGraph::new(),
+        ),
     )
-    .scan_for_tenant(tenant_id, 360, |_graph: MergedGraph| true)
-    .await; // and finally this blocks for another 10s
-
-    // Adding up all of the above, we have 40+s of blocking to get to this
-    // point. So the timeouts above need to each be at least 40s. See the
-    // warning in the docs for KafkaTopicScanner.scan().
-    // TODO: get those 40s back by running these all concurrently. This should
-    // allow us to reduce the timeouts to ~30s which will give significant gains
-    // in total test time.
+    .scan_for_tenant(tenant_id, 36, |_graph: MergedGraph| true)
```

Comment:
That number shows up elsewhere in these tests, as the file the test data come from has 36 lines. I guess we could load the data in _before_ setting up these scanners and use the actual number of events in the data as the number to scan for, but probably not worth it until that dataset changes

Reply:
Gotcha. We don't have to load stuff up beforehand, but it might be nice to better explain the meaning of the `36` in the context of this test. It wasn't immediately obvious what relationship the number `36` has to the test data.

Reply:
👍 ok I'll add a comment